src/settings.ts

```diff
@@ -36,6 +36,8 @@ export const SPARKLEVERSE_COMMUNITY_URL =
   "https://www.facebook.com/groups/sparkleverse/";
 export const CURRENT_TIME_IN_LOCATION = "Matong State Forest";
```

Comment:
[OOS] Another bit of outdated code we can 🔥

Reply:
Lets do this in https://github.com/sparkletown/sparkle/pull/833 .